docs/bgp.md

```diff
@@ -0,0 +1,161 @@
+# Configuration of BGP peering
+
+By default, a Calico-Docker network uses a full BGP mesh to distribute routes 
+between all of the Calico nodes.  This approach has scale limitations and so 
+various different topologies are recommended for scaling out a network.
+
+This document describes the commands available in calicoctl for managing BGP 
```

Comment:
Every time `calicoctl` or a `calicoctl command` is mentioned, you should stick backticks around it.